### PR TITLE
Interactive fixes

### PR DESCRIPTION
**Pull request by UnderGreen**

### CODE REVIEW

**Comment by a-palchikov** [comment]
Some description of changes is always welcome - so that the reviewer knows what to look for.

File: e2e/uimodel/defaults/defaults.go [hunk 1]
```diff
@@ -48,5 +48,5 @@ const (
 	OpsCenterDeleteSitePollInterval = 3 * time.Second
 
 	// BandwagonSubmitFormTimeout defines timeout for submit form request
-	BandwagonSubmitFormTimeout = 20 * time.Second
+	BandwagonSubmitFormTimeout = 300 * time.Second
```
[the remark this answers is not in the document]
Comment (UnderGreen) [reply e2e/uimodel/defaults/defaults.go]: In the case with anypoint user should wait a few minutes after submitting the form. I am not sure what they do in their bandwagon application and could they decrease that time.

File: e2e/uimodel/defaults/defaults.go [hunk 2]
```diff
@@ -6,7 +6,7 @@ const (
 	// AjaxCallTimeout specifies the amount of time needed to complete AJAX request
 	AjaxCallTimeout = 20 * time.Second
 	// AppLoadTimeout specifies the amount of time needed for web app to load
-	AppLoadTimeout = 20 * time.Second
+	AppLoadTimeout = 60 * time.Second
```
[the remark this answers is not in the document]
Comment (UnderGreen) [reply e2e/uimodel/defaults/defaults.go]: I think I changed it to check something and forgot to turn back.

File: e2e/framework/test_context.go [hunk 3]
```diff
@@ -215,14 +217,34 @@ type TestContextType struct {
...
-	Email        string `json:"email" yaml:"email" `
+	Organization string                `json:"organization" yaml:"organization" `
+	Username     string                `json:"username" yaml:"username" `
+	Password     string                `json:"password" yaml:"password" `
+	Email        string                `json:"email" yaml:"email" `
+	Extra        *BandwagonExtraConfig `json:"extra" yaml:"extra"`
 	RemoteAccess bool
 }
 
+// BandwagonExtraConfig defines configuration for extra bandwagon fields
```
[the remark this answers is not in the document]
Comment (UnderGreen) [reply e2e/framework/test_context.go]: These fields are actual only for Anypoint and I decided to create extra struct to store them.

File: e2e/framework/test_context.go [hunk 4]
```diff
@@ -284,6 +306,8 @@ type OnpremConfig struct {
 	ClusterAddress *ClusterAddress `json:"cluster_address" yaml:"cluster_address"`
 	// VariablesFile defines the path to file with custom terraform variables
 	VariablesFile string `json:"variables_file" yaml:"variables_file"`
+	// OnpremProvider specifies usage of onprem provider for installation
```
[the remark this answers is not in the document]
Comment (UnderGreen) [reply e2e/framework/test_context.go]: To set `--cloud-provider=generic` for installation. Or we fixed that and do need to define cloud provider for onprem installations on AWS?

File: e2e/framework/test_context.go [hunk 5]
```diff
@@ -221,28 +221,32 @@ type TestContextType struct {
...
-// BandwagonExtraConfig defines configuration for extra bandwagon fields
+// BandwagonExtraConfig defines configuration for extended bandwagon fields
 type BandwagonExtraConfig struct {
+	// PlatformDNS defines DNS address for accessing to platform
 	PlatformDNS string `json:"platform_dns" yaml:"platform_dns"`
-	NFSServer   string `json:"nfs_server" yaml:"nfs_server"`
-	NFSPath     string `json:"nfs_path" yaml:"nfs_path"`
+	// NFSServer defines address of NFS server used by platform
+	NFSServer string `json:"nfs_server" yaml:"nfs_server"`
+	// NFSPath identifies a file system exported by a remote NFS serve
```
Comment (a-palchikov) [inline e2e/framework/test_context.go]: serve**r**?

File: e2e/framework/test_context.go [hunk 6]
```diff
@@ -221,28 +221,32 @@ type TestContextType struct {
...
+	Email        string `json:"email" yaml:"email" `
+	// Extra defines extended configuration for bandwagon
 	Extra        *BandwagonExtraConfig `json:"extra" yaml:"extra"`
 	RemoteAccess bool
 }
 
-// BandwagonExtraConfig defines configuration for extra bandwagon fields
+// BandwagonExtraConfig defines configuration for extended bandwagon fields
 type BandwagonExtraConfig struct {
+	// PlatformDNS defines DNS address for accessing to platform
```
Comment (a-palchikov) [inline e2e/framework/test_context.go]: address of the DNS server?
nit: `accessing the platform`

**Pull request closed without merging**